code/src/pages/cart/CartPage.scss

```diff
@@ -0,0 +1,78 @@
+@import "@design-system";
+
+.spa-cart-page {
+  padding: spacing(10);
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    gap: spacing(5);
+  }
+
+  &__items,
+  &__order-summary {
+    padding: spacing(5);
+    border: 1px solid var(--spa-order-summary-border-color, $white);
+    border-radius: spacing(1);
+    background-color: var(--spa-order-summary-background-color, $white);
+  }
+
+  &__items {
+    width: 60%;
+
+    h1 {
+      margin-bottom: spacing(5);
+      font-size: spacing(8);
+      border-bottom: 1px solid
+        var(--spa-order-cart-title-border-color, $gray-light);
+      padding-bottom: spacing(3);
+    }
+  }
+
+  &__order-summary {
+    width: 35%;
+
+    h2 {
+      margin-bottom: spacing(5);
+      font-size: spacing(8);
+      border-bottom: 1px solid
+        var(--spa-order-summary-border-color, $gray-light);
+      padding-bottom: spacing(3);
+    }
+
+    .spa-order-summary__details {
+      p {
+        font-size: spacing(4);
+        margin: spacing(3) 0;
+      }
+
+      .spa-order-summary__row {
+        display: flex;
+        justify-content: space-between;
+
+        &.spa-order-summary__total-line {
+          font-weight: var(--spa-order-summary-font-weight, $fw-extra-bold);
+          font-size: spacing(5);
+          padding-top: spacing(7);
+          border-top: 1px solid
+            var(--spa-order-summary-line-border-color, $gray-light);
+        }
+      }
+
+      .spa-order-summary__underline-text {
+        text-decoration: underline;
+        margin: spacing(4) 0;
+      }
+
+      .spa-order-summary__total {
+        font-size: spacing(5);

```

Comment:
Also, typography variant

Reply:
> Also, typography variant

`variant` already exist
<img width="409" alt="image" src="https://github.com/idx-academy/spa-orders/assets/56305508/e6da311d-a96e-4793-9c07-87c727f6c3ee">

Reply:
use subtitle2 variant, it's exactly 20px font size

![image](https://github.com/idx-academy/spa-orders/assets/118522791/16658609-ac8b-4082-8f9c-3b861667a7b9)
![image](https://github.com/idx-academy/spa-orders/assets/118522791/5407057a-8b5f-4806-899a-bff6cf42265e)